titration/utils/devices/temperature_probe.py

```diff
@@ -22,3 +22,10 @@ def get_temperature(self):
 
     def get_resistance(self):
         return self.sensor.resistance
+
+    def read_temperature(self):
```

Comment:
What value does this function add?

Reply:
I see ... interfaces added this function. Lets address this in a future update. Please add an issue to remove this function?

Reply:
I'll add an issue